kind/values.yaml

```diff
@@ -14,7 +14,7 @@ orb:
       name: "orb-sinks"
       pullPolicy: "IfNotPresent"
       repository: "orbcommunity"
-      tag: "develop"
+      tag: "0.25.0-044b0c82"
```

Comment:
why is this hard coded?

Reply:
its wrong, i need to return to develop. forgot again... sorry. doing right now!